android/app/build.gradle

```diff
@@ -200,7 +197,7 @@ android {
             reset()
             enable enableSeparateBuildPerCPUArchitecture
             universalApk false  // If true, also generate a universal APK
-            include "armeabi-v7a", "x86"
+            include "armeabi-v7a", "arm64-v8a", "x86"
```

Comment:
so you still include x86 here after all?

Reply:
The configuration section isn't used right now since `enableSeparateBuildPerCPUArchitecture` is set to `false`:
https://github.com/status-im/status-react/blob/c2b55233225f6e4d59c9ca5174b5261cc1137fec/android/app/build.gradle#L97
And that means that this section, which normally would have caused Gradle to build a separate APK for each platform, does nothing.  I'm just adding `arm64-v8a` to match what we have set in `ndk.abiFilters`.

Now, since it seems to be necessary for dev builds, because devs use the emulator, it should be included by default, which is the `ndk.abiFilters` setting, but for pr, release, and nightly builds(except for e2e ones) it will be removed by setting the `NDK_ABI_FILTERS` env variable in `android.gradlke`.
See:
https://github.com/status-im/status-react/pull/8331/files#diff-921ddb8a284f72367049b92eff26b9b6R10